README.md

```diff
@@ -5,4 +5,26 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin integrates continuous development on [Kubernetes](https://www.kubernetes.io) using [Skaffold](https://skaffold.dev) into [Jetbrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm and WebStorm.
+
+**Note**: This plugin is still in development.
+
+## Features
+
+* One click deployment to Kubernetes clusters right from your IDE using kubectl, Helm, Google Cloud Build, Jib and Kanico using Skaffold integration with IDE.
+* Continuous development on Kubernetes. Watches the dependencies of your docker image or Jib Java project for changes, so that on any change, Skaffold builds and deploys your application to a Kubernetes cluster.
+* Automatic discovery and support for project with existing Skaffold configuration, in any language supported by your preferred Jetbrains IDE.
+* Initial support for Skaffold configuration files editing and smart templates.
+
+## Prerequisites and required dependencies
+
+This plugin relies on existing Kubernetes and container tools and libraries to provide rich Kubernetes experience built into your IntelliJ or other Jetbrains IDE. At the minimum, the following tools are expected to be installed and setup on your system and available in the system path:
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for working with Kubernetes clusters and managing Kubernetes deployments.
+* [Skaffold](https://skaffold.dev/docs/getting-started/) to support continuous development on a Kubernetes cluster, smart image building and tagging, and an array of supported deployment and build types.
+* [Docker](https://www.docker.com/) for building and pushing your container images. *Note*: Docker is optional if you are using [Jib to build container image](https://github.com/GoogleContainerTools/jib) together with a Maven/Gradle project for your Java IntelliJ IDE and push to a non-Docker container repository.
+* Configured Kubernetes cluster. It could be a cluster for local development, such as [Minikube](https://kubernetes.io/docs/setup/minikube/) or [Docker Kubernetes](https://docs.docker.com/docker-for-mac/kubernetes/) cluster, or remote cluster, such as [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) cluster. We recommend [Minikube](https://kubernetes.io/docs/setup/minikube/) cluster for local development.
+
+## Installing the plugin into your IDE
```

Comment:
what if we noted in this section that the plugin has not yet been published to the JetBrains plugin marketplace. Then provide steps for building the plugin from source and installing it in the IDE. The steps are roughly:

- Build the plugin (`./gradlew buildPlugin`)
- Find the distribution zip in the `build/distributions` folder
- In the plugin settings, choose to install a plugin from filesystem, and browse to the zip

Reply:
side note: if we decide that its worth our time, we could even automate via Kokoro copying the zip into a directory (at the root of the project for example) on every commit to master. This would avoid the initial build step.

Reply:
Excellent idea for automation prior to formal release channels available! We might even assign this to our new team members, would be a great help, WDYT?

Reply:
And before that is ready (`prerelease`), I will add mentioned steps to build and add the plugin from local disk - let me do it in the follow up PR to make it a bit cleaner to review, section by section, along with the screenshots, if you agree with this approach?

Reply:
Filed #139 for tracking.

Reply:
> And before that is ready (prerelease), I will add mentioned steps to build and add the plugin from local disk - let me do it in the follow up PR to make it a bit cleaner to review, section by section, along with the screenshots, if you agree with this approach?

 sgtm

Just to point out - the automation piece has some Kokoro components (updating some backend configs on Google3). I think it's still ok though as a starter.